dist/contour.js

```diff
@@ -3312,8 +3312,7 @@
                 // animationDirection: 'bottom-to-top',
                 marker: {
                     enable: true,
-                    size: 3,
-                    animationDelay: 0
```

Comment:
I like having `animationDelay` defined here, because this section lists all the possible options for the `marker` section and is self-documenting.

As an alternative you could consider initializing it with `null` and then changing the null check to be like https://github.com/forio/contour/blob/3297e03e0be64cbacef01eec6afc23a94fd0bab5/src/scripts/visualizations/line.js#L75 ` duration = options.chart.animations.duration != null ? options.chart.animations.duration : 400;` .